src/components/BackgroundImage.js

```diff
@@ -1,8 +1,16 @@
+//@flow
 import React, { Component } from 'react';
 import { Image, StyleSheet } from 'react-native';
 import Globals from '../Globals';
 
+import type { Children } from 'react';
+
+type Props = {
+  children?: Children
+}
+
 class BackgroundImage extends Component {
+  props: Props;
```

Comment:
do you like this https://flow.org/en/docs/frameworks/react/#toc-explicitly-specifying-react-component-generics?